addons/website_slides/views/website_slides.xml

```diff
@@ -99,7 +105,7 @@
                         </t>
                         <!-- Public user has not right to create/write on slide -->
                         <t t-if="channel.can_upload and not is_public_user">
-                            <a role="button" class="btn btn-primary oe_slide_js_upload" title="Upload Presentation" t-attf-channel_id="#{channel.id}" aria-label="Upload Presentation">
```

Comment:
Those seem unrelated to the task (and commit)

Reply:
Indeed, I ended up doing them on the other PR, they should disappear at the rebase (if merged second).